stable/yugaware/values.yaml

```diff
@@ -114,8 +114,14 @@ tls:
 ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/
 securityContext:
   enabled: false
-  fsGroup: 1001
-  runAsUser: 1001
+  ## fsGroup related values are set at the pod level.
+  fsGroup: 10001
+  fsGroupChangePolicy: "OnRootMismatch"
+  ## The following values are set for yugaware and prometheus
+  ## containers.
+  runAsUser: 10001
```

Comment:
minor: add a comment saying that if runAsUser is set to a diff value other than 10001, it will cause VM universe workflows to fail.